.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,12 @@
+---
```

Comment:
This sort of change needs to happen in `inst/templates/tidy-issue.md`. Then we could apply the revised function to usethis itself, to change its issue template(s).